.gitignore

```diff
@@ -9,3 +9,5 @@
 /plugins/ZynAddSubFx/zynaddsubfx/doc/Makefile
 /plugins/ZynAddSubFx/zynaddsubfx/doc/gen/Makefile
 /data/locale/*.qm
+
+.vscode
```

Comment:
Should this be `/.vscode` since it's a directory?

Reply:
I would think it should be ``.vscode/*``. ``/.vscode`` seems like it's could be referring to the root directory.

Reply:
All paths in a .gitignore file are relative to the location of the .gitignore file itself.

It looks like I was wrong in my first comment: If it is `/.vscode`, it will still match either a file or directory - it just matches a ".vscode" file or directory in the repo's root directory.

And according to the [documentation](https://git-scm.com/docs/gitignore#_examples):
> The pattern `foo/` will match a directory `foo` and paths underneath it, but will not match a regular file or a symbolic link `foo`

So unless the .vscode directory can be anywhere in the repo, not just the root directory, `/.vscode/` might be what we want.

Reply:
Starting with commit ea9c28fd it's `/.vscode/` to signify that it has to be a directory in the root directory that's called `.vscode`.